### PR TITLE
Initial work on context menu for additional card functionality

### PR DESCRIPTION
**Pull request by jleach**
Ground work for the delete and card details context menu. I wasn't table to finish the feature so this just adds the ellipsis menu and shows and alert explaining the feature is not yet implemented. This PR also fixed the workflow for doing the iOS build.

Screen Shot 2021-09-28 at 6 18 00 PM
.

### CODE REVIEW

**Review by amanji — COMMENTED** [review]
The structure I've been following is Views -> whole screens, Components -> smaller components that screens consume.

File: src/views/credential/ContextMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  background-color: ${theme.colors.white};
+  opacity: 1;
+  width: 200px;
+  border-radius: 10px;
+  elevation: 5;
+  z-index: 5;
+`;
+
+export const ContextMenu: React.FC<IModalMenuProps> = () => {
+  const [modalIsVisible, setModalIsVisible] = useState(false);
```
Comment (amanji) [inline src/views/credential/ContextMenu.tsx:33]: Is it better to use state like this rather than props?

**Review by amanji — COMMENTED** [review]
Some minor clarifications

File: src/views/credential/ContextMenu.tsx (hunk 1 above)
Comment (jleach) [reply src/views/credential/ContextMenu.tsx:33]: Yes, props. This is some draft code that is inactive in this PR. I'm trying a few different ways in the background.

**Comment by jleach** [comment]
> The structure I've been following is Views -> whole screens, Components -> smaller components that screens consume.

Noted. I'll follow the same. Most of the code in this PR is disabled; only the `Alert` view is accessible. Once I have a working menu I'll clean it up as you mention above.

**Review by amanji — APPROVED** [review]

**Pull request merged**